app/javascript/data/advancedFilterOptions.js

```diff
@@ -424,8 +424,9 @@ export const advancedFilterOptions = [
       {
         name: 'cohort-type',
         title: 'cohort type',
-        type: 'select',
+        type: 'multi',
         options: [
+          '<any>',
```

Comment:
so I really don't like the `<any> and `` options because they're contradictory to the whole purpose behind a multi select.  see the screenshot below, users can select blank, any and any subset of options.  it's pretty confusing.  is there a reason why we can't have no options selected be `<any>` and not support blank?  we don't support blank in any other multi selects advanced filters

<img width="348" alt="Screen Shot 2021-11-17 at 8 13 08 AM" src="https://user-images.githubusercontent.com/35042815/142207431-e16d2823-faf7-4259-b157-d758b152a9e8.png">

Reply:
started a thread here: https://mitrecorp.slack.com/archives/C029JM3PFC4/p1637163798089200

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/9224bca33be59de6ff1b6a3827c57c21f0a27a87#diff-9609a958c89218f92ffdf7d0c6635f81734d4951c7261a249248114a23fcd24dL429